packages/sdk/tsconfig.web.json

```diff
@@ -0,0 +1,22 @@
+{
+  "extends": "./tsconfig.json",
+  "include": ["src/indexWeb.ts", "src/initWeb.ts", "../shared/src/shared/**/*", "../crypto/src/crypto/**/*"],
+  "exclude": [
+    "**/*.test.ts",
+    "node_modules",
+    "scripts"
+  ],
+   "compilerOptions": {
+    "declaration": true,
+    "outDir": "dist/web",
+    "module": "esnext",
+    "moduleResolution": "bundler",
+    "plugins": [
+      // Transform paths in output .js files
```

Comment:
Probably a silly question, but does this still work even though comments are not allowed in JSON files? :smile:

Reply:
I can tell you more! :) Even trailing commas work, the reason is that typescript uses it's own parser :P